datasets/hover/README.md

```diff
@@ -0,0 +1,163 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+- found
+languages:
+- en
+licenses:
+- cc-by-sa-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- text-retrieval
+task_ids:
+- fact-checking-retrieval
+---
+
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://hover-nlp.github.io/
+- **Repository:** https://github.com/hover-nlp/hover
+- **Paper:** https://arxiv.org/abs/2011.03088
+- **Leaderboard:** https://hover-nlp.github.io/
+- **Point of Contact:** [More Information Needed]
+
+### Dataset Summary
+
+[More Information Needed]
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+```
+DatasetDict({
+    train: Dataset({
+        features: ['id', 'uid', 'claim', 'supporting_facts', 'label', 'num_hops', 'hpqa_id'],
+        num_rows: 18171
+    })
+    validation: Dataset({
+        features: ['id', 'uid', 'claim', 'supporting_facts', 'label', 'num_hops', 'hpqa_id'],
+        num_rows: 4000
+    })
+    test: Dataset({
+        features: ['id', 'uid', 'claim', 'supporting_facts', 'label', 'num_hops', 'hpqa_id'],
+        num_rows: 4000
+    })
+})
+```
+
+### Data Fields
+
+A sample training set is provided below
+
+```
+{'id': 14856, 'uid': 'a0cf45ea-b5cd-4c4e-9ffa-73b39ebd78ce', 'claim': 'The park at which Tivolis Koncertsal is located opened on 15 August 1843.', 'supporting_facts': [{'key': 'Tivolis Koncertsal', 'value': 0}, {'key': 'Tivoli Gardens', 'value': 1}], 'label': 'SUPPORTED', 'num_hops': 2, 'hpqa_id': '5abca1a55542993a06baf937'}
+```
+
+Please note that in test set sentence only id, uid and claim are available.
```

Comment:
Can you mention that you replace the missing labels in the test set with a default value ? (probably -1 if you switch to ClassLabel)